src/diff/props.js

```diff
@@ -90,12 +90,17 @@ export function setProperty(dom, name, value, oldValue, isSvg) {
 		else name = name.slice(2);
 
 		if (!dom._listeners) dom._listeners = {};
+		if (!useCapture && Array.isArray(value)) {
```

Comment:
What happens here when the user changes from passive to active? We would need to re-register the event-handler which might be unexpected that we aren't doing that. It did make me wonder whether the `-capture` would be a good fit for the `-passive` as well. The only other options are `once` and `signal` which I am not sure would fit here reliably....

Reply:
@JoviDeCroock According to MDN only the `capture` option is checked for matching event listeners. See: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#matching_event_listeners_for_removal